.gitignore

```diff
@@ -234,6 +234,9 @@ cython_debug/
 
 # Sphinx documentation
 
+# Editors
```

Comment:
Don't add these entries to the repo because they are not related to the project.

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c